docs/howto.md

```diff
@@ -188,6 +188,94 @@ future. Do note that by separating the credentials in the secret, the password
 is never directly accessed by the operator itself.
 
 
+# Use a cluster internal share as persistent volume (without auto-provisioning)
+
+Kubernetes does not by default support mounting `smb` shares.  For now
```

Comment:
SMB is an acronym and should be in all upper case. This is also the convention followed in the rest of the Document for describing the protocol.